src/nginxconfig/templates/domain_sections/https.vue

```diff
@@ -206,6 +243,14 @@ THE SOFTWARE.
             default: true,
             enabled: true,
         },
+        http3: {
+            default: false,
+            enabled: true,
+        },
+        portReuse: {
+            default: true,
```

Comment:
```suggestion
            default: false,
```
We probably don't want reuseport to be on by default

Reply:
However, it might be interesting to look at showing a warning if http3 is enabled without reuseport, but as reuseport is a setting separate from http2/3 usage, I think we'll want it always available as an option but therefor not on by default.

Reply:
Nvm, ignore me on this, trying to do this is probably going to result in a more complex UI that'll confuse folks.